webapp/components/test-search.html

```diff
@@ -81,27 +81,23 @@
         this.onBlur = this.handleBlur.bind(this);
       }
 
-      connectedCallback() {
-        super.connectedCallback();
+      ready() {
```

Comment:
The reason I moved this is because it doesn't work as-is. `shadowRoot` is null until `ready` is called.
This setup isn't correct anyway; the input element belongs to the custom element (with a 1:1 pairing), so we don't need to remove the event listener at all.